server/common/common-dao-gen/src/main/resources/sqlmap/FlywaySchemaHistoryDOMapper.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`server/common/common-dao-gen/src/main/resources/sqlmap` should only caontains auto-generated *Mapper.xml